dask/dataframe/io/hdf.py

```diff
@@ -385,7 +389,10 @@ def read_hdf(pattern, key, start=0, stop=None, columns=None,
         lock = get_scheduler_lock()
 
     key = key if key.startswith('/') else '/' + key
-    paths = sorted(glob(pattern))
+    if isinstance(pattern, str):
+        paths = sorted(glob(pattern))
+    else:
+        paths = pattern
```

Comment:
Ah, I see. Thanks for the information, I wasn't aware of the functions in `dask.bytes`!